public/app/features/expressions/components/Threshold.tsx

```diff
@@ -0,0 +1,148 @@
+import { css } from '@emotion/css';
+import React, { FC, FormEvent } from 'react';
+
+import { GrafanaTheme2, SelectableValue } from '@grafana/data';
+import { ButtonSelect, InlineField, InlineFieldRow, Input, Select, useStyles2 } from '@grafana/ui';
+import { EvalFunction } from 'app/features/alerting/state/alertDef';
+
+import { ClassicCondition, ExpressionQuery, thresholdFunctions } from '../types';
+
+interface Props {
+  labelWidth: number;
+  refIds: Array<SelectableValue<string>>;
+  query: ExpressionQuery;
+  onChange: (query: ExpressionQuery) => void;
+}
+
+const defaultThresholdFunction = EvalFunction.IsAbove;
+
+export const Threshold: FC<Props> = ({ labelWidth, onChange, refIds, query }) => {
```

Comment:
I'm wondering if we could re-use, some way, code from `Condition.tsx`?

Reply:
There's probably some code we can re-use; I tried to avoid creating additional abstractions in this PR that I might have change later on. Any specific parts of the code you think we could abstract?

Reply:
@gillesdemey  
For example, this could be a sub common component , reused in both `Condition.tsx` and  in `Treshold.tsx`
<img width="2406" alt="Screenshot 2022-09-19 at 12 37 42" src="https://user-images.githubusercontent.com/33540275/190999998-a1d47683-70f1-4e95-8b58-81025cd1c298.png">

alongside this method (that could be in this new common component, so we don't repeat the same code)
<img width="2325" alt="Screenshot 2022-09-19 at 12 45 12" src="https://user-images.githubusercontent.com/33540275/191001164-49b4dad1-aaf8-4131-a96a-12321536a158.png">